product-service/tests/getProductsById.test.js

```diff
@@ -0,0 +1,26 @@
+import { handler as getProductsById } from "../handlers/getProductsById";
+import producsList from "../productsList.json";
+
+test("Getting product by correct id", async () => {
```

Comment:
Small advice. It's better to mock internal functions such as `getAllProducts` and `getOneProduct`. Otherwise, when you switch to real DB implementation, you will also have to modify these tests.